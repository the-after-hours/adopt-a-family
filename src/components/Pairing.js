// Page to connect familes with donators

import React from 'react';

const Pairing = (props) => (
  <div>
    <h1>Pairing page</h1>
    <p>A collection of potential people that you can pair with</p>
    <ul>
      <li>
        personA{' '}
        <span style={{ align: 'right', marginLeft: '3em' }}>
          <button>PAIR</button>
        </span>
      </li>
      <li>
        personB{' '}
        <span style={{ align: 'right', marginLeft: '3em' }}>
          <button>PAIR</button>
        </span>
      </li>
      <li>
        personC{' '}
        <span style={{ align: 'right', marginLeft: '3em' }}>
          <button>PAIR</button>
        </span>
      </li>
      <li>
        personD{' '}
        <span style={{ align: 'right', marginLeft: '3em' }}>
          <button>PAIR</button>
        </span>
      </li>
    </ul>
  </div>
);

export default Pairing;
