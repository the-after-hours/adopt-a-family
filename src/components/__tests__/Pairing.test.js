import React from 'react';
import Pairing from '../Pairing.js';
import renderer from 'react-test-renderer';

describe('Pairing', () => {
  test('Pairing snapshot', () => {
    const component = renderer.create(<Pairing />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});