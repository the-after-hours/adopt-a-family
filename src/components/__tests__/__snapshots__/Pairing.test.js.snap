// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pairing Pairing snapshot 1`] = `
<div>
  <h1>
    Pairing page
  </h1>
  <p>
    A collection of potential people that you can pair with
  </p>
  <ul>
    <li>
      personA
       
      <span
        style={
          Object {
            "align": "right",
            "marginLeft": "3em",
          }
        }
      >
        <button>
          PAIR
        </button>
      </span>
    </li>
    <li>
      personB
       
      <span
        style={
          Object {
            "align": "right",
            "marginLeft": "3em",
          }
        }
      >
        <button>
          PAIR
        </button>
      </span>
    </li>
    <li>
      personC
       
      <span
        style={
          Object {
            "align": "right",
            "marginLeft": "3em",
          }
        }
      >
        <button>
          PAIR
        </button>
      </span>
    </li>
    <li>
      personD
       
      <span
        style={
          Object {
            "align": "right",
            "marginLeft": "3em",
          }
        }
      >
        <button>
          PAIR
        </button>
      </span>
    </li>
  </ul>
</div>
`;
